README.md

```diff
@@ -1 +1,125 @@
-# Todo-List
\ No newline at end of file
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+
+# 📖 [My Microverse TO-DO-LIST PROJECT] <a name="about-project"></a>
+
+> Todo List is a site that displays the list of the days tasks.
+
+## 🛠 Built With
+
+- HTML
+- CSS
+- JavaScript
+
+
+
+## Key Features
+
+-  Displays list.
+-  Items can be added to the list.
+-  Items can be deleted.
+-  All selected Item can be marked as complete
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](Comming soon)
```

Comment:
- [ ] [OPTIONAL I kindly suggest that you format this better. you do not need to make this a link because that space within the parathesis is for a link. For example:

```
## Live Demo
Coming Soon 🚀 
```